templates/lxc-busybox.in

```diff
@@ -237,6 +238,7 @@ lxc.cap.drop = sys_module mac_admin mac_override sys_time
 
 lxc.mount.auto = cgroup:mixed proc:mixed sys:mixed
 lxc.mount.entry = shm dev/shm tmpfs defaults 0 0
+lxc.mount.entry = mqueue dev/mqueue mqueue defaults 0 0
```

Comment:
This should use `defaults,create=dir`

Reply:
So, we can remove mkdir of /dev/shm in :

  fstree="\
${rootfs}/selinux \
${rootfs}/dev \
${rootfs}/home \
${rootfs}/root \
${rootfs}/etc \
${rootfs}/etc/init.d \
${rootfs}/bin \
${rootfs}/usr/bin \
${rootfs}/sbin \
${rootfs}/usr/sbin \
${rootfs}/proc \
${rootfs}/sys \
${rootfs}/mnt \
${rootfs}/tmp \
${rootfs}/var/log \
${rootfs}/var/run \
${rootfs}/dev/pts \
${rootfs}/dev/shm \  <-------------------------------- Useless ?
${rootfs}/lib \
${rootfs}/usr/lib \
${rootfs}/lib64 \
${rootfs}/usr/lib64"